### PR TITLE
Dx 802 document msisd email checkout

### PR DESCRIPTION
**Pull request by marinafrydenb**
documented this in checkout

### CODE REVIEW

File: _includes/callback-reference.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ When a change or update from the back-end system are made on a payment or
 transaction, Swedbank Pay will perform a callback to inform the payee (merchant)
 about this update.
 
-* Setting a `callbackUrl` in the HTTP `POST` API is optional, but highly
+* Setting a `callbackUrl` in the HTTP `POST` API is **mandatory** and highly
```
Comment (NullableInt) [inline _includes/callback-reference.md]: Don't you already have a pull request for this? #320

Comment (marinafrydenb) [reply _includes/callback-reference.md]: ops this branch should not have this

File: _includes/payment-order-purchase.md [hunk 2]
```diff
@@ -126,6 +126,10 @@ Content-Type: application/json
...
+|          | └─➔&nbsp;`email`            | `string`      | The e-mail address of the payer.                                                                                                                                                             |
+|          | └─➔&nbsp;`msisdn`            | `string`      | Number will be prefilled on payment page, if valid. The mobile number must have a country code prefix and be 8 to 15 digits in length.                                                                                                                                                             |
+|          | └─➔&nbsp;`workPhoneNumber`            | `string`      | Optional (increased chance for frictionless flow if set).                                                                                    |
```
Comment (asbjornu) [inline _includes/payment-order-purchase.md]: Is this correct information about `workPhoneNumber`? I don't see any mention of it in DX-802.

File: _includes/payment-order-purchase.md [hunk 3]
```diff
@@ -126,6 +126,10 @@ Content-Type: application/json
...
 |          | └➔&nbsp;`payer`                   | `object`     | The `payer` object containing information about the payer relevant for the payment order.                                                                                                                                                                                                                                                                                                                     |
 |   ︎︎︎    | └─➔&nbsp;`consumerProfileRef`     | `string`     | The consumer profile reference as obtained through [initiating a consumer session][initiate-consumer-session].                                                                                                                                                                                                                                                                                                |
+|          | └─➔&nbsp;`email`            | `string`      | The e-mail address of the payer.                                                                                                                                                             |
```
Comment (asbjornu) [inline _includes/payment-order-purchase.md]:
```suggestion
|          | └─➔&nbsp;`email`            | `string`      | The e-mail address of the payer. Will be used to prefill the Checkin as well as on the payer's profile, if not already set.                                                                                                                                                             |
```

File: _includes/payment-order-purchase.md [hunk 4]
```diff
@@ -126,6 +126,10 @@ Content-Type: application/json
...
+|          | └─➔&nbsp;`msisdn`            | `string`      | Number will be prefilled on payment page, if valid. The mobile number must have a country code prefix and be 8 to 15 digits in length.                                                                                                                                                             |
```
Comment (asbjornu) [inline _includes/payment-order-purchase.md]:
```suggestion
|          | └─➔&nbsp;`msisdn`            | `string`      | The mobile phone number of the Payer. Will be prefilled on Checkin page and used on the payer's profile, if not already set. The mobile number must have a country code prefix and be 8 to 15 digits in length.                                                                                                                                                             |
```

**Comment by asbjornu** [comment]
Please open a new PR that deals with my review comments.

**Pull request merged**